scripts/upload-secrets/upload_secrets.rb

```diff
@@ -8,10 +8,18 @@
 require "yaml"
 require "tempfile"
 
-def get_secret(path)
+def get_secret(path, *default_out)
```

Comment:
This could be a named argument instead, eg

```
def get_secret(path, default=nil)
```

then your else becomes

```
abort out if default.nil?

out.to_s
```